docs/platforms/apple/guides/ios/build-distribution/auto-update.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Auto-Update SDK
+sidebar_title: Auto-Update SDK
+sidebar_order: 10
+description: Enable automatic update checks and installations for internal iOS builds using the Sentry Auto-Update SDK.
+---
+
+<Include name="feature-available-for-user-group-early-adopter" />
+
+<Alert level="warning">
+
+The Auto-Update SDK is designed for **internal builds only** and should never be used in production builds distributed through the App Store.
+
+</Alert>
+
+The Sentry Auto-Update SDK enables your internal iOS builds to automatically check for and install newer versions distributed through Sentry's Build Distribution. This is particularly useful for distributing nightly, alpha, or beta builds to your internal teams. It is not required to use the Sentry crash reporting SDK to use the iOS Auto-Update SDK.
+
+## Pre-requisites
+
+The SDK can only be installed using Swift Package Manager.
+
+You'll also need an [internal integration token](#create-an-integration-token) with Build Distribution permissions.
+
+<Include name="build-distribution/create-integration-token" />
+
+## Installation
+
+Add a dependency on the `SentryDistribution` target contained in the sentry-cocoa package (https://github.com/getsentry/sentry-cocoa)
+
+```swift {filename:Package.swift}
+.package(url: "https://github.com/getsentry/sentry-cocoa", from: "{{@inject packages.version('sentry.cocoa') }}"),
+
+.target(name: "MyTarget", dependencies: ["SentryDistribution"]),
+```
+
+## Usage
+
+Use the SDK by calling `Updater.checkForUpdate(params: )`. In addition to the access token, provide your Sentry org and project slug in the CheckForUpdateParams. For example:
```

Comment:
I'm not very familiar with xcode but is there a way to provide a script to xcode that would create a file in the app that would automatically pass these things in?